src/apic-extension/azext_apic_extension/aaz/latest/__init__.py

```diff
@@ -4,3 +4,7 @@
 #
 # Code generated by aaz-dev-tools
 # --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
```

Comment:
Minor: the auto generated changes looks good to me, but I don't know why it's not included in previous version. Can anyone share some backgrounds?

Reply:
The style-suppress default code line is added into aaz-dev tool in June 2022.